app/assets/javascripts/initializers/initializeBaseUserData.js

```diff
@@ -1,15 +1,17 @@
-'use strict';
 
-/* global userData, filterXSS */
+
+/* global filterXSS */
```

Comment:
eslint 6 in code climate was complaining about `userData` already being considered global. The rest below is prettier in action.